Hi @BillyONeal 

Thanks for this PR.

It seems that `signalrclient ` still requires for `websockets `.
Since `signalrclient ` depends on `cpprestsdk`.

`C:\vsts\_work\3\s\buildtrees\signalrclient\src\.0.0-beta1-3b1429b26e\src\signalrclient\default_websocket_client.h(26): error C2039: 'websockets': is not a member of 'web'`

So we might also need to update `signalrclient ` to fix this regression.

--- a/ports/signalrclient/CONTROL
+++ b/ports/signalrclient/CONTROL
@@ -1,5 +1,5 @@
 Source: signalrclient
-Version: 1.0.0-beta1-7
-Build-Depends: cpprestsdk, openssl
+Version: 1.0.0-beta1-8
+Build-Depends: cpprestsdk[default-features,websockets], openssl
 Homepage: https://github.com/aspnet/SignalR-Client-Cpp
 Description: C++ client for SignalR.

